example/t3dn_bip_example/ops.py

```diff
@@ -110,18 +112,107 @@ def execute(self, context: bpy.types.Context) -> set:
         return {'FINISHED'}
 
 
+icon_items: List[Tuple[str, str, str]] = []
```

Comment:
Why is that global and local to get_icon_items?

Reply:
Its because of that weird enum bug right? Mmm...

Reply:
In that case, please add a comment to that variable.

Reply:
Ah and by the way, this will fail you when you have multiple instances of bip_example_template_icon_view. As you are clearing the list.

Reply:
I shall add the comment, I've also moved it to a module named `enum_lists.py`, so we know exactly what it's doing.

Reply:
I've had issues with storing the list as a global, which were resolved by storing the list as an item of a global dictionary... take that as you will.

Reply:
These issues we're with KIT OPS specifically.